<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortBridge</class>
 <widget class="QWidget" name="PortBridge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="bridge_gb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-10</y>
     <width>561</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="src_db">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>20</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Source Port</string>
    </property>
    <widget class="QSpinBox" name="src_port">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>7001</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="dst_gb">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Destination IP/Port</string>
    </property>
    <widget class="QSpinBox" name="dst_port">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>2222</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="dst_ip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.0.100</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="start_stop">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
